reorder_python_imports/main.py

```diff
@@ -335,8 +337,14 @@ def main(argv=None):
     args = parser.parse_args(argv)
 
     retv = 0
-    for filename in args.filenames:
-        contents = io.open(filename).read()
+
+    if not isinstance(args.filenames, list):
+        files = [args.filenames]
+    else:
+        files = args.filenames
+
+    for input_file in files:
+        contents = six.u(input_file.read())
```

Comment:
I think this'll break on non-ascii files in python2 (I should add one as a test?)

the lhs handles this using the `io` module